package.json

```
@@ -10,10 +10,26 @@
     "lint": "eslint *.js",
     "start": "node api.js",
     "deploy": "now --alias",
-    "precommit": "npm run lint",
-    "textlint:ja": "textlint --config .textlintrc-ja ja/**/*.md",
+    "textlint:ja": "textlint --config ja/.textlintrc ja/**/*.md",
     "textlint:ja:fix": "yarn textlint:ja --fix --format diff"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "linters": {
+      "*.js": [
+        "eslint --fix",
+        "git add"
+      ],
+      "ja/**/*.md": [
+        "textlint --config ja/.textlintrc --fix --format diff",
```

```suggestion
        "textlint --config ja/.textlintrc --fix --format diff ja/**/*.md",
```
For Windows OS. 
対象とするファイルを記載しないとWindowsではチェックが走りませんでした

---

Thank you for your check!!

Then we checked and understood the following:

- It had no effect on Windows even with "ja/**/*.md"
- It occur no error on Windows even without "ja/**/*.md"

Therefore there is no harm in merge this pull request. :smile: